stdlib/public/Concurrency/Task.swift

```diff
@@ -145,12 +156,10 @@ extension Task where Failure == Never {
   /// creating the task with the correct priority,
   /// depending on the executor's scheduling details.
   ///
-  /// ### Cancellation
-  /// The task this refers to may check for cancellation, however
-  /// since it is not-throwing it would have to handle it using some other
-  /// way than throwing a `CancellationError`, e.g. it could provide a neutral
-  /// value of the `Success` type, or encode that cancellation has occurred in
-  /// that type itself.
+  /// Because this method is nonthrowing,
+  /// if the task checks for cancellation,
+  /// it needs to handle cancellation using an approach like returning `nil`
+  /// instead of throwing an error.
```

Comment:
I suggest wording this differently, this isn't as much about this computed property being not throwing, but the entire `Task` type this is declared on is `Error == Never`, so it is more about:

> Since this Task is nonthrowing, this value cannot throw and if the task wanted to react to cancellation it could do so in some other way, e.g. by returning `nil` but it cannot throw.

?

Reply:
Rewrote in 946c72cbc73:

> Tasks that never throw an error can still check for cancellation,
> but they need to use an approach like returning `nil`
> instead of throwing an error.

I think this version is a little more direct too.

Reply:
Sounds good :)